cuda/io/CMakeLists.txt

```diff
@@ -2,16 +2,17 @@ set(SUBSYS_NAME cuda_io)
 set(SUBSYS_PATH cuda/io)
 set(SUBSYS_DESC "Point cloud CUDA IO library")
 set(SUBSYS_DEPS cuda_common io common)
+set(EXT_DEPS openni)
```

Comment:
SUBSYS_EXT_DEPS in keeping with the theme?

Reply:
Ah, yes.